exercises/dominoes/tests/dominoes.rs

```diff
@@ -6,9 +6,9 @@ type Domino = (usize, usize);
 
 #[derive(Debug)]
 enum CheckResult {
-    GotInvalid,             // chain returned None
+    GotInvalid, // chain returned None
     Correct,
-    ChainingFailure(Vec<Domino>), // failure to match the dots at the right side of one domino with
+    ChainingFailure(Vec<Domino>), /* failure to match the dots at the right side of one domino with */
```

Comment:
The answer is probably somewhere in here: https://github.com/rust-lang-nursery/rustfmt/blob/master/src/comment.rs

But I haven't spotted it.

I'm going to update this with a change that should be easier to read and won't be changed by rustfmt.